processingchain/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/Workflow.groovy

```diff
@@ -7,6 +7,155 @@ import org.orbisgis.datamanager.JdbcDataSource
+                            inseeCodes << row.insee_com
+                        }
+
+                        //Let's run the BDTopo process for each insee code
+                        def prepareBDTopoData = ProcessingChain.PrepareBDTopo.prepareBDTopo()
+                        int nbAreas = inseeCodes.size();
+                        info "$nbAreas areas will be processed"
+                        inseeCodes.eachWithIndex { code, index->
+                         if(prepareBDTopoData.execute([datasource                 : datasource,
+                                                    tableIrisName              : 'IRIS_GE', tableBuildIndifName: 'BATI_INDIFFERENCIE',
+                                                    tableBuildIndusName        : 'BATI_INDUSTRIEL', tableBuildRemarqName: 'BATI_REMARQUABLE',
+                                                    tableRoadName              : 'ROUTE', tableRailName: 'TRONCON_VOIE_FERREE',
+                                                    tableHydroName             : 'SURFACE_EAU', tableVegetName: 'ZONE_VEGETATION',
+                                                    tableImperviousSportName   : 'TERRAIN_SPORT', tableImperviousBuildSurfName: 'CONSTRUCTION_SURFACIQUE',
+                                                    tableImperviousRoadSurfName: 'SURFACE_ROUTE', tableImperviousActivSurfName: 'SURFACE_ACTIVITE',
+                                                    distBuffer                 : 500, expand: 1000, idZone: code,
+                                                    hLevMin                    : 3, hLevMax: 15, hThresholdLev2: 10
+                        ])){
+
+                             String buildingTableName = prepareBDTopoData.getResults().outputBuilding
+
+                             String roadTableName = prepareBDTopoData.getResults().outputRoad
+
+                             String railTableName = prepareBDTopoData.getResults().outputRail
+
+                             String hydrographicTableName = prepareBDTopoData.getResults().outputHydro
+
+                             String vegetationTableName = prepareBDTopoData.getResults().outputVeget
+
+                             String zoneTableName = prepareBDTopoData.getResults().outputZone
+
+                             IProcess geoIndicators = GeoIndicators()
+                             if (!geoIndicators.execute( datasource          : datasource,           zoneTable       : zoneTableName,
+                                     buildingTable       : buildingTableName,    roadTable       : roadTableName,
+                                     railTable           : railTableName,        vegetationTable : vegetationTableName,
+                                     hydrographicTable   : hydrographicTableName,indicatorUse    : indicatorUse,
+                                     svfSimplified       : svfSimplified,        prefixName      : prefixName,
```

Comment:
It seems that even for two different inseecode, you will have the same output table names. Thus you will remove almost all resulting tables from the datasource. Is this made on purpose ?
If not I would propose to add the InseeCode in the prefixName such as prefixName = PrefixName +"_" + codeInsee.

Reply:
Done with the prefixname =  zone_codeinsee_